lib/modules/manager/github-actions/extract.ts

```diff
@@ -246,6 +334,17 @@ function extractWithYAMLParser(
     };
 
     for (const step of coerceArray(job?.steps)) {
+      if (step.uses) {
+        for (const action of communityActions) {
+          if (action.use.test(step.uses)) {
+            const val = Result.parse(step, action.schema).unwrapOrNull();
```

Comment:
this hides all errors, you need to log them as debug, so users are aware of wrongly configured actions.

Reply:
I don't think we need error log for case actions without `with`? This warning are useless to users.

Reply:
> I don't think we need error log for case like actions without `with`? This warning are useless to users IMO

Reply:
And how to do this when I rewrite it with a zod union?